### PR TITLE
add Binary instance for Hashed

### PR DESCRIPTION
**Pull request by minad**

### CODE REVIEW

File: src/Data/Binary/Orphans.hs [hunk 1]
```diff
@@ -41,7 +41,7 @@ import qualified Data.CaseInsensitive as CI
 import qualified Data.Fixed as Fixed
 import qualified Data.HashMap.Lazy as HM
 import qualified Data.HashSet as HS
-import           Data.Hashable (Hashable)
+import           Data.Hashable
```
Comment (phadej) [inline src/Data/Binary/Orphans.hs]: I insist on explicit import lists.

```hs
import           Data.Hashable (Hashable)
#if MIN_VERSION_hashable(1,2,5)
import           Data.Hashable (Hashed, hashed, unhashed)
#endif
```

File: src/Data/Binary/Orphans.hs [hunk 2]
```diff
@@ -243,7 +249,7 @@ instance Binary a => Binary (NE.NonEmpty a) where
 instance Binary m => Binary (Semigroup.WrappedMonoid m) where
   get = fmap Semigroup.WrapMonoid get
   put = put . Semigroup.unwrapMonoid
-
```
Comment (phadej) [inline src/Data/Binary/Orphans.hs:246]: :+1:

**Review by phadej — CHANGES_REQUESTED** [review]
One nitpick, otherwise LGTM

**Comment by phadej** [comment]
qualified is fine as well. Will merge when travis is green.

**Review by phadej — APPROVED** [review]

**Comment by minad** [comment]
cool thx

**Pull request merged**